site/content/docs/5.0/examples/pricing/pricing.css

```diff
@@ -1,3 +1,7 @@
+body {
+  background-image: linear-gradient(180deg, #eee, #fff 100px, #fff);
```

Comment:
check orange design

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380